class ProviderError(Exception):
    pass


class NotFound(ProviderError):
    pass
